aiogram/client/session/base.py

```diff
@@ -193,6 +193,8 @@ def prepare_value(
             return value
         if isinstance(value, Default):
             default_value = bot.default[value.name]
+            if isinstance(default_value, TelegramObject):
+                default_value = default_value.model_dump(warnings=False)
```

Comment:
This line should be covered by tests.

More elegant way that is move this condition into one level with other conditions at this function instead of adding one more level.